Clients/src/presentation/components/ProjectCard/ProgressBar/index.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Progress bar display component for calculating the status of completed actions in fields.
+ *
+ * @component
+ * @param {ProgressBarProps} props - The properties for the ProgressBar component.
+ * @param {string} props.progress - The progress indicator as a fraction.
+ *
+ * @returns {JSX.Element} The rendered ProgressBar component.
+ */
+
+import { Box } from "@mui/material";
+import { FC } from "react";
+
+interface ProgressBarProps {
+    progress: string
+}
+
+const ProgressBar: FC<ProgressBarProps> = ({progress}) => {
+    const progressCount = (progressString: string): number => {
+        const [completed, total] = progressString.split('/').map(Number);
+        if (isNaN(completed) || isNaN(total) || total === 0) {
+          console.error(`Invalid progress string: ${progressString}`);
+          return 0;
+        }
+        return completed / total;
+      };
+    const widthValue = progressCount(progress) * 100;
+      
+    return (
+        <Box sx={{ width: "100%", backgroundColor: "#EAECF0", borderRadius: 2, height: 8, position: "relative" }}>
```

Comment:
Thank you Yevheniia, I noticed you implemented a custom progress bar, however, since we're already using MUI, we could take advantage of the built-in <LinearProgress /> component. It comes with some handy features and customization options that could simplify the code and make maintenance a bit easier in the future.

Reply:
Please check this Slider component from MUI and see if you can manipulate and bind it according to our needs.
[MUI Slider](https://mui.com/material-ui/react-slider/)
Check to see if you can remove its handle and do not hesitate to reach out to me if you need any help.
Cheers 🍻